app/lib/rendering/glyphBasics.es6.js

```diff
@@ -97,66 +97,53 @@ function()
           , i,l
           ;
 
+        // Actually, all points have controls. We don't have to draw
+        // lines. We should make a CPS value if we want to draw a
+        // point as a line segment point
+        segmentType = 'curve';
+
         pen.beginPath();
         // first draw the right side
-        for(i=0,l=points.length;i<l;i++) {
+        for(i=0,l=points.length; i<l; i++) {
             point = model.getComputedStyle(points[i].right);
-            // Actually, all points have controls. We don't have to draw
-            // lines. We should make a CPS value if we want to draw a
-            // point as a line segment point
-            if(true /* always curve */) {
```

Comment:
It's on my list to implement exporting lines, by the way. However, keep it like this if it helps you :-) I think the implementation will become more comlex anyways.